docs/docs/cli-reference/package-init.md

```diff
@@ -1,14 +1,19 @@
-# kurtosis package init
-This command initializes the current directory to be a [Kurtosis package][package] by creating a [`kurtosis.yml` file][kurtosis-yml] with the given package name.
+---
+title: package init
+sidebar_label: package init
+slug: /package-init
+---
+
+The `package init` command converts the current directory into a [Kurtosis package][package] by generating a new [`kurtosis.yml`][kurtosis-yml] file using the given package name.
```

Comment:
it will create the 'kurtosis.yml' and the 'main.star' file by default now.

Reply:
```suggestion
The `package init` command converts the current directory into a [Kurtosis package][package] by generating a new [`kurtosis.yml`][kurtosis-yml] file and a `main.star` file using the given package name, if they do not already exist.
```